x-pack/solutions/security/plugins/security_solution/public/timelines/components/timeline/query_bar/eql/index.tsx

```diff
@@ -70,7 +70,9 @@ export const EqlQueryBarTimeline = memo(({ timelineId }: { timelineId: string })
 
   const { newDataViewPickerEnabled } = useEnableExperimental();
 
-  const { dataView: experimentalDataView, status } = useDataView(SourcererScopeName.timeline);
+  const { dataViewSpec: experimentalDataView, status } = useDataViewSpec(
```

Comment:
Same here for example, `experimentalDataView` is used only in one place right below, why not just keeping the word `dataViewSpec`? Or `experimentalDataViewSpec` if you prefer?